snapshot-labs/snapshot-webhook
# feat: Add walletconnect notify  provider for web3Inbox

Continued from https://github.com/snapshot-labs/snapshot-webhook/pull/114

Credit to @devceline and @chris13524


### TODO:
- [ ] Before merging add the following env (others are already added)
```
WALLETCONNECT_NOTIFICATION_TYPE=
```

### How to test: 
- Contact @ChaituVR  for env values
- Subscribe to your space by clicking the bell icon
![image](https://github.com/snapshot-labs/snapshot-webhook/assets/15967809/b24c56ff-1403-46b8-a6b0-be85dbcc53b1)
- Go to https://app.web3inbox.com/ and subscribe to Snapshot App
- Once you run `yarn dev` and a new proposal is started on space you subscribed, you should get notification on web3inbox
- You should see a notification like 
![image](https://github.com/snapshot-labs/snapshot-webhook/assets/15967809/db03c2e4-72c0-4a63-aaa7-e31f27eeab6a) and on click, it should redirect to proposal
